### ISSUE TITLE
Broken nightly build

### ISSUE THREAD

**Issue by vallsv** [silx-kit/silx#702]
The silx nightly build was broken since 1 month, while the mail sent was "Nightly silx: Success".

It would be good to fix the process.

```
/include/median_filter.hpp:93: void median_filter(const T*, T*, int*, int*, int, int, int, int, bool) [with T = short int]: Assertion `x_pixel_range_min <= x_pixel_range_max' failed.
Aborted
E: pybuild pybuild:256: test: plugin custom failed with: exit code=134: env PYTHONPATH=/home/tester/silx/package/silx-0.5.0a0/.pybuild/pythonX.Y-dbg_2.7/build WITH_QT_TEST=False python2.7-dbg run_tests.py
dh_auto_test: pybuild --test -i python{version}-dbg -p 2.7 -s custom --test-args=env PYTHONPATH={build_dir} WITH_QT_TEST=False {interpreter} run_tests.py --dir . returned exit code 13
debian/rules:39: recipe for target 'override_dh_auto_test' failed
make[1]: *** [override_dh_auto_test] Error 13
make[1]: Leaving directory '/home/tester/silx/package/silx-0.5.0a0'
debian/rules:13: recipe for target 'build' failed
make: *** [build] Error 2
dpkg-buildpackage: error: debian/rules build gave error exit status 2
Build failed, please investigate ...
skipping directory silx-0.5.0a0
.
```

**Comment by vallsv** [comment]
I fix the build-deb8 scripts to return the error status. It should fix the problem.

**Comment by vasole** [comment]
I think this should be closed, shouldn't it?

**Comment by t20100** [comment]
Closing, fixed by #815
